aws/table_aws_iam_user.go

```diff
@@ -483,3 +505,21 @@ func userMfaStatus(_ context.Context, d *transform.TransformData) (interface{},
 
 	return false, nil
 }
+
+func userMfaDevices(_ context.Context, d *transform.TransformData) (interface{}, error) {
```

Comment:
Is this what we normally call transform functions? I would expect some verb, e.g., `get`, `list`, `format`, in the function name, as I'm not sure what output I'd expect just from the function name.